test/test-group.js

```diff
@@ -302,6 +302,28 @@ describe('Groups', function () {
       });
     });
 
+    // it('should remove last member from group and delete group ', function () {
+    //   User.findById(user2.id)
```

Comment:
user2 needs to be updated because we add user 2 to group 2 in test above, but then since user2 here doesnt have that update, req.user.isMemberOf(group2) doesnt work on backend...